### ISSUE TITLE
Allow BPMN message name to be hard coded in Sink connector

### ISSUE THREAD

**Issue by berndruecker** [zeebe-io/kafka-connect-zeebe#20]
In some situation records on one Kafka topic should always sent into Zeebe with the same message name. Currently you have to extract the message name itself from the data of the record.

Current config
```
    "message.path.messageName": "$.variablesAsMap.messageName",
    "message.path.correlationKey": "$.variablesAsMap.key",
    "message.path.variables": "$.variablesAsMap"
```

I want to do this

```
    "message.path.messageName": "messageName",
    "message.path.correlationKey": "$.variablesAsMap.key",
    "message.path.variables": "$.variablesAsMap"
```

Or maybe we need to rename the attribute into e.g.
```
    "message.path.messageNameExpression": "$.variablesAsMap.messageName",     
    "message.path.messageName": "messageName",
```
Which I think is inline to what we do at other places in Zeebe.

And I actually wondered why the attribute is called "message.path" - what's the "path" in it? @npepinpe - probably you know?

**Comment by npepinpe** [comment]
For JSONPath, but admittedly it probably wasn't necessary to add it there. I think we should focus on migrating to SMT support instead of using JSONPath.

**Comment by berndruecker** [comment]
I tend to agree (even if this is a quite big refactoring and still will require some thinking around the message name makes total sense to be hardcoded

**Comment by npepinpe** [comment]
I would propose that we allow all values to be hardcoded as well, for example for fan out behaviour you might want the correlation key to be hardcoded to an empty string.
